README.md

```diff
@@ -1140,7 +1140,7 @@ _Related: [Note-taking & Editors](#note-taking--editors), [Wikis](#wikis)_
 Tools and software to help with education and learning.
 
 - [Canvas LMS](https://www.instructure.com/canvas/) - Canvas is the trusted, open-source learning management system (LMS) that is revolutionizing the way we educate. ([Demo](https://canvas.instructure.com/register), [Source Code](https://github.com/instructure/canvas-lms)) `AGPL-3.0` `Ruby`
-- [Chamilo LMS](https://chamilo.org/) - Chamilo LMS allows you to create a virtual campus for the provision of online or semi-online training. ([Source Code](https://github.com/chamilo/chamilo-lms)) `GPL-3.0` `PHP`
+- [Chamilo LMS](https://chamilo.org/en/) - Chamilo LMS allows you to create a virtual campus for the provision of online or semi-online training. ([Source Code](https://github.com/chamilo/chamilo-lms)) `GPL-3.0` `PHP`
```

Comment:
please revert this, the website will redirect visitors to the correct page based on their browser language settings